### PR TITLE
Add semantic attributes section to python manual instrumentation doc

### PR DESCRIPTION
**Pull request by cartermp**
Contributes to https://github.com/open-telemetry/opentelemetry.io/issues/1539

Preview: https://deploy-preview-1703--opentelemetry.netlify.app/docs/instrumentation/python/manual/#add-semantic-attributes

### CODE REVIEW

File: content/en/docs/instrumentation/python/manual.md [hunk 1]
```diff
@@ -155,6 +155,29 @@ current_span.set_attribute("operation.name", "Saying hello!")
...
+
+```python
+from opentelemetry import trace
+from opentelemetry.semconv.trace import SpanAttributes
+
+// ...
+
+current_span = trace.get_current_span()
+rollspan.set_attribute(SpanAttributes.HTTP_METHOD, "GET")
+rollspan.set_attribute(SpanAttributes.HTTP_URL, "www.gooddeals.com")
```
Comment (svrnm) [inline content/en/docs/instrumentation/python/manual.md:178]: Replace that URL with something boring (example.com) or some URL affiliated with otel / cncf (opentelemetry.io, cncf.io, ...)

**Review by svrnm — CHANGES_REQUESTED** [review]

File: content/en/docs/instrumentation/python/manual.md (hunk 1 above)
Comment (cartermp) [reply content/en/docs/instrumentation/python/manual.md:178]: Ah, bummer, but where will people get their good deals!? 😛

**Comment by cartermp** [comment]
@svrnm should be good to go now

File: content/en/docs/instrumentation/python/manual.md [hunk 2]
```diff
@@ -155,6 +155,29 @@ current_span.set_attribute("operation.name", "Saying hello!")
...
+
+```python
+from opentelemetry import trace
+from opentelemetry.semconv.trace import SpanAttributes
+
+// ...
+
+current_span = trace.get_current_span()
+rollspan.set_attribute(SpanAttributes.HTTP_METHOD, "GET")
+rollspan.set_attribute(SpanAttributes.HTTP_URL, "https://opentelemetry.io/")
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/manual.md:178]: Either remove the `current_span` or update the below lines to use it.

```suggestion
current_span.set_attribute(SpanAttributes.HTTP_METHOD, "GET")
current_span.set_attribute(SpanAttributes.HTTP_URL, "https://opentelemetry.io/")
```

**Review by srikanthccv — APPROVED** [review]

**Review by svrnm — APPROVED** [review]
lgtm

**Pull request merged**